src/components/NavigationBar/index.tsx

```diff
@@ -1,99 +1,75 @@
 import Link from "next/link";
-import Image from "next/image";
-import { useState } from "react";
+import { useEffect, useState } from "react";
 import ACMLogo from "public/assets/ACMWhiteLogo.png";
-import CloseMenuButton from "public/assets/closemenubutton.svg";
-import MenuButtonIcon from "public/assets/menubutton.svg";
+import s from "src/components/NavigationBar/Navbar.module.scss";
+import { Size, useWindowSize } from "src/utils";
 
 const navLinks = [
+  { to: "#contact", text: "Contact" },
   { to: "/sponsor", text: "Sponsor" },
   { to: "/communities", text: "Communities" },
   { to: "/about", text: "About Us" },
 ];
 
 const NavigationBar: React.FC = () => {
-  const [menuState, setMenuState] = useState(false);
+  const size: Size = useWindowSize();
 
-  const toggleMenu = () => setMenuState(!menuState);
+  const [menuOpen, setMenuOpen] = useState(false);
+  const [mobile, setMobile] = useState(false);
+  const toggleMenu = () => setMenuOpen(!menuOpen);
+
+  // Switch to mobile less than 920px
+  useEffect(() => {
+    setMobile(size.width < 920);
+  }, [size]);
+
+  // If they go back to desktop size, don't keep the menu open
+  useEffect(() => {
+    if (!mobile) setMenuOpen(false);
+  }, [mobile]);
 
   return (
-    <div className="fixed-nav">
-      <div className="navigation-bar">
-        <ul>
-          <li className="navigation-bar__logo">
-            <Link href="/">
-              <a>
-                {/* image breaking the website right now */}
-                <img src={ACMLogo.src} alt="ACM Logo" />
-                <p>at UCSD</p>
-              </a>
-            </Link>
-          </li>
-          <li className="navigation-bar__login">
-            <Link href={"https://members.acmucsd.com/login"}>
-              <a>Member Login</a>
-            </Link>
-          </li>
-          <li className="navigation-bar__nav-button">
-            <Link href="#contact">
-              <a>Contact</a>
-            </Link>
-          </li>
+    <div className={s.navbarWrapper}>
+      <div className={s.navbar}>
+        {/* Navbar ACM Logo */}
+        <div className={s.left}>
+          <Link href="/">
+            <a>
+              <img src={ACMLogo.src} alt="ACM Logo" />
+              <p>at UCSD</p>
+            </a>
+          </Link>
+        </div>
+
+        {/* Desktop Nav Links */}
+        <div className={`${s.right} ${mobile && s.hidden}`}>
           {navLinks.map((link, key) => (
-            <li key={key} className="navigation-bar__nav-button">
-              <Link href={link.to}>
-                <a>{link.text}</a>
-              </Link>
-            </li>
+            <Link key={key} href={link.to}>
```

Comment:
the links are rendering in the reverse order rn - can you make it so the links render as

About Us
Communities 
Sponsor
Contact
Login

rn it's this
<img width="352" alt="Screen Shot 2022-02-28 at 3 39 18 PM" src="https://user-images.githubusercontent.com/9388431/156077236-f33f7b82-faec-4896-a2ce-ba6bc622c4cf.png">

it should be this order instead


<img width="258" alt="Screen Shot 2022-02-28 at 3 39 51 PM" src="https://user-images.githubusercontent.com/9388431/156077263-4605a140-5a98-4e63-a7a5-a2f18dbc0652.png">